### ISSUE TITLE
Feature Request: Add Duration Button in the Graph Window

### ISSUE THREAD

**Issue by aliraeisdanaei** [vmiklos/plees-tracker#490]
Currently when you want to view the graph for different durations, you have to go into the settings and set the duration, then go back to the graph menu. 

It would really help to have that button open on the graph menu that can switch the graphs back and forth easily.

**Comment by aliraeisdanaei** [comment]
Yes exactly it will only affect the graph window. I am interested in trying to do this.

**Comment by vmiklos** [comment]
Great, looking forward to your PR. :-)

**Comment by phpmoli** [comment]
Agree with this, the Duration change is the only option in the Settings, that has daily use, it would be better out of there.
Also, if #605 is accepted, there would be no need for it at all, no? Or are there other places, Duration setting is altering behaviour?

The kebab menu in the Graphs screen could also get an easier cycle option.
